functions/go/apply-setters/README.md

```diff
@@ -6,27 +6,23 @@
 
 Update the field values parameterized by setters.
 
-##### Setters Definition
+#### Definitions
 
-Setters serve as parameters for customizing field values.
-Setters are a safer alternative to parameterize field 
-values as they have the context of the structured data.
+**Setters**: Setters serve as parameters for customizing field values.
+Setters are a safer option to parameterize field values as they have 
+the context of the structured data.
```

Comment:
Rephrase. The reason is that the resource is still a valid resource adhering to the KRM schema. Whereas other types of templating are no longer valid resources.

Reply:
Updated.